pkg/data/management/role_data.go

```diff
@@ -85,6 +85,8 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 	userRole.addNamespacedRule("cattle-global-data").addRule().apiGroups("").resources("secrets").verbs("create")
 
 	rb.addRole("User Base", "user-base").
+		// TODO security assessment - standard permissions for regular users, on their tokens
```

Comment:
This was more about me asking if the set of verbs is ok in terms of security.
Reworked the comment in commit 57d5046e36730211ae7a31e854e639417aa782e0.
Because these verbs should be ok, and `manage-token` is the one to be careful with.